modules/installation-complete-user-infra.adoc

```diff
@@ -130,6 +139,51 @@ command.
 +
 If the pod logs display, the Kubernetes API server can communicate with the
 cluster machines.
+ifdef::ibm-z,ibm-power[]
+. For an installation with FCP, additional steps are required to enable multipathing. 
+.. To enable multipathing on master nodes, apply the following machine config file: 
++
+[source,yaml]
+--
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: "master"
+  name: 99-master-kargs-mpath
+spec:
+  kernelArguments:
+    - 'rd.multipath=default'
+    - 'root=/dev/disk/by-label/dm-mpath-root'
+--
+
+.. To enable multipathing on worker nodes, apply the following machine config file: 
++
+[source,yaml]
+--
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: "worker"
+  name: 99-master-kargs-mpath
+spec:
+  kernelArguments:
+    - 'rd.multipath=default'
+    - 'root=/dev/disk/by-label/dm-mpath-root'
+--
```

Comment:
ifeval::["{context}" == "installing-ibm-power"]

To perform the actual MPIO fail over check, leverage the bootlist command to update the boot device list with alternate logical device names.

The following command displays a boot list and it designates the possible boot devices for when the system is booted in normal mode.

```
$ bootlist -m normal -o
sda
```

To update the boot list for Normal boot mode, update additional alternate device names that you would like to add as follows:

```
$ bootlist -m normal -o /dev/sdc /dev/sdd /dev/sde
sdc
sdd
sde
```

Next node reboot will boot from the alternate device registered in the normal boot device list, when the previous/original boot disk path is down

endif::[]

ifeval::["{context}" == "installing-restricted-networks-ibm-power"]

To perform the actual MPIO fail over check, leverage the bootlist command to update the boot device list with alternate logical device names.
The following command displays a boot list and it designates the possible boot devices for when the system is booted in normal mode.

```
$ bootlist -m normal -o
sda
```
To update the boot list for Normal boot mode, update additional alternate device names that you would like to add as follows:

```
$ bootlist -m normal -o /dev/sdc /dev/sdd /dev/sde
sdc
sdd
sde
```
Next node reboot will boot from the alternate device registered in the normal boot device list, when the previous/original boot disk path is down

endif::[]


@SNiemann15

Reply:
This statement can be removed:

```
To perform the actual MPIO failover check, leverage the bootlist command to update the boot device list with alternate logical device names.
```

Also this section seems to have the same text twice.